lib/elixir/lib/enum.ex

```diff
@@ -3849,12 +3849,10 @@ defmodule Enum do
   Returns the `enumerable` with each element wrapped in a tuple
   alongside its index or according to a given function.
 
-  May receive a function or an integer offset.
-
-  If an `offset` is given, it will index from the given offset instead of from
+  If an `offset` integer is given, it will index from the given offset instead of from
   zero.
 
-  If a `function` is given, it will index by invoking the function for each
+  If an `offset` function is given, it will index by invoking the function for each
```

Comment:
Given that the name of the documented arg is `fun_or_offset`, this feels a bit confusing (especially since the function is not an offset)